lib/util/accessibility-routing.js

```diff
@@ -2,17 +2,5 @@
  * Determine if an itinerary has accessibility scores
  */
 export const itineraryHasAccessibilityScores = (itinerary) => {
```

Comment:
Because we are looking for an accessiblity score for all legs, we should rename this function to `itineraryHasAccessibilityScoresOnAllLegs` or `hasAccessibilityScoresOnAllLegs`.

Reply:
Since the previous version of this function was just checking for itinerary scores on any of the legs, I repurposed it to check for the itinerary score itself! Which should make the name of this function still make sense but let me know if you disagree.

Reply:
Sounds good. But it should be singular (`itineraryHasAccessibilityScore`) since you are only checking one score!

Reply:
Good point, changed in [ab6574e](https://github.com/opentripplanner/otp-react-redux/pull/1092/commits/ab6574ee9fe335fddcb9706508ebf9933ebc74a2)!